### ISSUE TITLE
conts.js -> rename it to const.template.js

### ISSUE THREAD

**Issue by Sirhexalot** [tagip/forest-guard#22]
Hi,

is it possible to conts.js -> rename it to const.template.js.

Because if you dont work with env vars your settings are overwritten on each pull.

**Comment by mikaoelitiana** [comment]
That's correct, I should do some refactoring inside to improve the code in general.
